public/sendme.sh

```diff
@@ -27,16 +27,16 @@ release_target_url=$(
     sed -re 's/.*: "([^"]+)".*/\1/' \
 )
 
-if [[ "$release_target_url" =~ \.zip$ ]]; then
+if [ "${release_target_url##*.}" = "zip" ]; then
```

Comment:
While this is POSIX compliant, as you mentioned in [#317](https://github.com/n0-computer/iroh.computer/pull/317/files/9de205f2f3ad359c97bcb713d03c032bac8f635c#r2040366016), I believe that `grep` should be used instead for a more robust matching:
```suggestion
if echo "$release_target_url" | grep -q '\.zip$'; then
```

Reply:
I see no reason for `echo` with `grep` instead of shell-builtin `test` implementation.

Reply:
JFYI: I tested the `[ "${release_target_url##*.}" = "zip" ]` with `dash` which is strictly POSIX-complaint and it works w/o hiccups.